### DISCUSSION TITLE
守护程序：检测到转发规则顺序错误，修改插件防火墙规则...

### DISCUSSION THREAD

**Discussion by ssgnh** [vernesong/OpenClash#3082] [Q&A]
刚启动openclash后，openclash界面的延迟检测都很好，但过一会，国内的就延迟飙升，打开网站有时很慢，查看日志，守护程序：检测到转发规则顺序错误，修改插件防火墙规则...，测试了很多次，都是刚启动程序时，延迟和网站打开速度都很快，一出现守护程序检测错误后，延迟特别不稳定，尤其是国内的
![无标题](https://user-images.githubusercontent.com/39829308/223458066-e94723b7-d5d2-483b-94c2-66d4ca86a312.png)

**Comment by ssgnh** [comment]
说明一下，用的爱快做主路由，openwrt做从路由，openwrt上的ipv6都关闭了

**Reply by phoetry** [reply]
一样的问题，但改为TUN模式后，似乎便不再报错
